Libraries/pbxbuild/Sources/Tool/AssetCatalogResolver.cpp

```diff
@@ -28,19 +31,41 @@ resolve(
     pbxsetting::Environment const &baseEnvironment,
     std::vector<Phase::File> const &inputs) const
 {
+    std::vector<std::string> absoluteInputPaths;
+    std::vector<std::string> assetPaths;
+    std::vector<std::string> stickerPackStrings;
+    for (Phase::File const &input : inputs) {
+        if (input.fileType()->identifier() == "text.plist.strings") {
+            std::string strings;
```

Comment:
Nit: you could allocate your `strings` variable and then `reserve()` the amount of space you're going to need before doing the concatenations. That'll avoid a bunch of reallocations. This might not even be a hot path, so feel free to ignore the suggestion.